karapace/karapace_all.py

```diff
@@ -27,6 +29,11 @@ def main() -> int:
 
     logging.basicConfig(level=logging.INFO, format=DEFAULT_LOG_FORMAT_JOURNAL)
     logging.getLogger().setLevel(config["log_level"])
+    if config.get("access_logs_debug") in [True, 1, "on", "true", "True"]:
```

Comment:
This should be simply:

```suggestion
    if config.get("access_logs_debug") in True:
```

Environment variables are parsed here:

https://github.com/aiven/karapace/blob/ac9587338bb4d844293fcfbcc3401b3b5f763491/karapace/config.py#L116